frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,254 @@
+import { execSync } from 'child_process';
+import { ExpectedConditions as until, browser } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import {
+  goToOperatorHub,
+  installOperator,
+  InstallCluster,
+  searchInOperatorHub,
+  uninstallOperator,
+  closeOperatorInstall,
+  storageClusterLink,
+  btnCreate,
+  selectWorkerRows,
+  status,
+  NS,
+  operatorsPage,
+  ocsOperator,
+  ocsOperatorStatus,
+  filterInput,
+  getDataFromRowAndCol,
+  podNameFilter,
+  goToStorageClasses,
+  checkIfClusterIsReady,
+  getPodData,
+  getPodPhase,
+  getPodName,
+} from '../views/subscriptionFlow.view';
+
+const installer = new InstallCluster(NS);
+
+const ocsDeploy = `
+---
+apiVersion: v1
```

Comment:
Creation of namespace and labeling it is a user workflow (part of installation flow), bypassing it by applying yaml is not something I agree to. This should probably be done as a pre-req for Subscription Scenario.